README.md

```diff
@@ -1,14 +1,192 @@
 [![License](http://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html) [![Build Status](https://travis-ci.org/simp/pupmod-simp-logrotate.svg)](https://travis-ci.org/simp/pupmod-simp-logrotate) [![SIMP compatibility](https://img.shields.io/badge/SIMP%20compatibility-4.2.*%2F5.1.*-orange.svg)](https://img.shields.io/badge/SIMP%20compatibility-4.2.*%2F5.1.*-orange.svg)
```

Comment:
Should update the compatibility versions

Reply:
I agree, but I wasn't sure what should go there.  I also wasn't sure if that should be part of this PR or if it should be submitted separately.

Reply:
@silug Just remove the compatibility badges for now.

Reply:
@silug I took care of it.